14:13 < mst__> _rene_, the toolkit unoapi tests are known to be flaky (in some 
               system dependent way) e.g. on the Win@6 tinderbox it always 
               crashes

--- a/toolkit/Module_toolkit.mk-old	2012-09-05 14:24:27.129693868 +0200
+++ b/toolkit/Module_toolkit.mk	2012-09-05 14:24:39.521755312 +0200
@@ -35,8 +35,8 @@
     Package_util \
 ))
 
-$(eval $(call gb_Module_add_subsequentcheck_targets,toolkit,\
-    JunitTest_toolkit_unoapi \
-))
+#$(eval $(call gb_Module_add_subsequentcheck_targets,toolkit,\
+#    JunitTest_toolkit_unoapi \
+#))
 
 # vim: set noet sw=4 ts=4:
